File: packages/marie-plugin-document-extraction/tests/fixtures/sample.cs
using System.Collections.Generic;

namespace Invoicing
{
    public class InvoiceParser
    {
        public double TotalAmount(List<double> amounts)
        {
            double total = 0;
            foreach (var amount in amounts)
            {
                total += amount;
            }
            return total;
        }
    }
}

File: packages/marie-plugin-document-extraction/tests/fixtures/sample.rs
use std::collections::HashMap;

pub struct InvoiceParser {
    currency: String,
}

impl InvoiceParser {
    pub fn parse(&self, path: &str) -> HashMap<String, f64> {
        let _ = path;
        HashMap::new()
    }
}

pub fn total_amount(amounts: &[f64]) -> f64 {
    amounts.iter().sum()
}
